apis/v1alpha2/grpcroute_types.go

```diff
@@ -272,15 +272,15 @@ type GRPCRouteRule struct {
 // For example, the match below will match a gRPC request only if its service
 // is `foo` AND it contains the `version: v1` header:
 //
-// ```
-// matches:
-// - method:
-//    type: Exact
-//    service: "foo"
-//   headers:
-//   - name: "version"
-//     value "v1"
-// ```
+//	```
+//	matches:
+//	- method:
+//	   type: Exact
+//	   service: "foo"
+//	  headers:
+//	  - name: "version"
+//	    value "v1"
+//	```
```

Comment:
Unfortunately this didn't translate well to the generated reference docs 😢. https://deploy-preview-1416--kubernetes-sigs-gateway-api.netlify.app/references/spec/#gateway.networking.k8s.io%2fv1alpha2.GRPCRouteMatch. Not sure what the solution is, there likely is some combination of indentation that works well for most. I have noticed that inline yaml examples only seem to fail gofmt on top level structs/comments were the actual `//` is not indented.

Reply:
Unfortunate, but a good find with the top level comments. Is that a viable way forward? Otherwise I suggest closing the pr because it doesn't seem to be possible to move forward as is.